feat(root): create component gallery, adjust styling on Components page (#669)
mi6/ic-design-system: src/content/structured/components/index.mdx

---
path: "/components"

date: "2023-11-07"

title: "Components"

subTitle: "Components are reusable user interface elements. These are the basic building blocks of designs create consistent, accessible and usable experiences across our missions, departments and partners."

contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/components/index.mdx"
---

<ComponentGallery />

> Can’t find what you’re looking for? We are expanding the Design System and UI Kit. You can check out the [roadmap](/community/roadmap) for what's coming. If you can help add to the system you can [read more on how to contribute](/community/contribute).

Reviewer: ```suggestion
Can’t find what you’re looking for? We are expanding the Design System and UI Kit. You can check out the [roadmap](/community/roadmap) for what's coming. 

Why not help add to the design system? Here's some more information on [how to contribute](/community/contribute).
```

Author: I'd be happy with this change if @DBD324 is? :)

---
path: "/components"

date: "2023-11-08"

title: "Components"

> subTitle: "Components are reusable user interface elements. These are the basic building blocks of designs create consistent, accessible and usable experiences across our missions, departments and partners."

Author: If @DBD324 is happy with this, I am happy too :)
